src/test/optimizer/strategy/predicate_reordering_rule_test.cpp

```diff
@@ -205,9 +205,12 @@ TEST_F(PredicateReorderingTest, PredicatesWithMultipleOutputs) {
      *    \           /
      *     Predicate_b
      *         |
+     *     Predicate_c
+     *         |
      *       Table
      *
-     * predicate_a should come before predicate_b - but since Predicate_b has two outputs, it can't be reordered
+     * Predicate_a should come before Predicate_b - but since Predicate_b has two outputs, it can't be reordered.
```

Comment:
not your fault, but "should come" is misleading. No, it should not, because that would be wrong.